account_financial_report/static/src/js/report_action.esm.js

```diff
@@ -1,40 +1,40 @@
-/** @odoo-module **/
-import {ReportAction} from "@web/webclient/actions/reports/report_action";
-import {patch} from "web.utils";
-import {useEnrichWithActionLinks} from "./report.esm";
+// /** @odoo-module **/
```

Comment:
This feature is lost due to this commented:

![imagen](https://github.com/OCA/account-financial-reporting/assets/7165771/64ae38fb-b32b-420c-8b8e-8100cc3debe9)

Reply:
`patch(ReportAction.prototype, "account_financial_report.ReportAction", {`
This line is causing error. How to do it in 17.0?

Reply:
```
patch(ReportAction.prototype, {
    setup() {
```
`        this._super.apply(this, arguments);`
TypeError: Cannot read properties of undefined (reading 'apply')

`        super.apply(...arguments);`
TypeError: (intermediate value).apply is not a function

Reply:
`        // super.apply(arguments);`
TypeError: Cannot read properties of undefined (reading 'el')

Reply:
`        super.setup(...arguments);`
This worked

Reply:
@pedrobaeza Could you check if last changes solves pending issues? Thanks